backend/dal/Helpers/Extensions/EntityEntryExtensions.cs

```diff
@@ -17,23 +17,12 @@ public static class EntityEntryExtensions
         /// <param name="entry"></param>
         public static void UndoAppCreatedEdits(this EntityEntry entry)
         {
-            if (entry.Entity is IDisableBaseAppEntity entity)
```

Comment:
Fixes the context checking for the wrong entity type. BaseAppEntity contains the timestamps

Reply:
This is not correct, IDisableBaseAppEntity inherits from IBaseAppEntity.

Reply:
Then what would happen if the entity does not have a disabled entry, has timestamps?

Reply:
oh, sorry I thought this was on line 81.